src/m3/functions.ts

```diff
@@ -40,35 +53,53 @@ export const relationsOf = (metamodelElement: MetamodelElement): Link[] =>
         : []
 
 
-export type Comparer<T> = (l: T, r: T) => number
+export const containeds = (thing: M3Concept): M3Concept[] => {
```

Comment:
Here a comment could help to clarify the role of this. Should this be renamed to `containedElements`?

Reply:
I'll add a comment. “elements” is not entirely appropriate (I think) as `EnumerationLiteral`s are not `MetamodelElement`s.